hadoop-project/pom.xml

```diff
@@ -1723,17 +1723,6 @@
           </exclusion>
         </exclusions>
       </dependency>
-      <dependency>
-        <groupId>org.apache.hbase</groupId>
-        <artifactId>hbase-server</artifactId>
-        <version>${hbase.version}</version>
-        <exclusions>
-          <exclusion>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
-          </exclusion>
-        </exclusions>
-      </dependency>
```

Comment:
I was unable to find any similar changes to this in 5c348c41ab8ddb81146355570856e61e8d129a1e (the backport commit of #3904) or other related PRs that were merged:
```
> git diff 4ba463069bd apache/branch-3.3  -- hadoop-project/pom.xml | cat
diff --git a/hadoop-project/pom.xml b/hadoop-project/pom.xml
index 8dc1862ed57..39dac39eac7 100644
--- a/hadoop-project/pom.xml
+++ b/hadoop-project/pom.xml
@@ -218,6 +218,7 @@
     <nodejs.version>v12.22.1</nodejs.version>
     <yarnpkg.version>v1.22.5</yarnpkg.version>
     <apache-ant.version>1.10.11</apache-ant.version>
+    <jmh.version>1.20</jmh.version>
   </properties>

   <dependencyManagement>
@@ -1603,6 +1604,16 @@
          </exclusion>
        </exclusions>
      </dependency>
+      <dependency>
+        <groupId>org.openjdk.jmh</groupId>
+        <artifactId>jmh-core</artifactId>
+        <version>${jmh.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.openjdk.jmh</groupId>
+        <artifactId>jmh-generator-annprocess</artifactId>
+        <version>${jmh.version}</version>
+      </dependency>
      <dependency>
        <groupId>org.apache.curator</groupId>
        <artifactId>curator-test</artifactId>
```
Why is this change necessary / how is it related to the backport?

Reply:
Ok, that was caused by HADOOP-18088 in 160b6d106.